integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -260,6 +260,35 @@ void testListTable() {
     Assertions.assertThrows(NoSuchNamespaceException.class, () -> listTableNames("not_exists_db"));
   }
 
+  @Test
+  public void testAlterTableSetProperty() {
+    String tableName = "testSetProperty";
+    dropTableIfExists(tableName);
+
+    createSimpleTable(tableName);
+    SparkTableInfo oldProperties = getTableInfo(tableName);
+    Assertions.assertFalse(oldProperties.getTableProperties().containsKey("key1"));
+
+    sql(String.format("alter table %s SET TBLPROPERTIES('key1'='value1')", tableName));
```

Comment:
could you test set multi properties like set key1 and key2, check both exists, remove one, check only the left key exists?

Reply:
fixed